integration/command.go

```diff
@@ -481,42 +481,54 @@ func (c *Command) Stop(t *testing.T) {
 	c.Started = false
 }
 
-// BusyboxPodRepeatCommand returns a Command that creates a pod and runs
-// "cmd" each 0.1 seconds inside the pod.
-func BusyboxPodRepeatCommand(namespace, cmd string) *Command {
-	cmdStr := fmt.Sprintf("while true; do %s ; sleep 0.1; done", cmd)
-	return BusyboxPodCommand(namespace, cmdStr)
-}
+// PodCommand returns a Command that starts a pid with a specified image, command and args
+func PodCommand(podname, image, namespace, command, commandArgs string) *Command {
+	cmdLine := ""
+	if command != "" {
+		cmdLine = fmt.Sprintf("\n    command: %s", command)
+	}
+
+	commandArgsLine := ""
+	if commandArgs != "" {
+		commandArgsLine = fmt.Sprintf("\n    args:\n    - %s", commandArgs)
+	}
```

Comment:
Not related to this PR: I wonder if we should consider using the k8s-apiserver directly instead of invoking kubectl. It seems the things we have are getting complicated and using the K8s API could simplify that.

Reply:
Tracking this in https://github.com/inspektor-gadget/inspektor-gadget/issues/1112